myria/relation.py

```diff
@@ -0,0 +1,71 @@
+from dateutil.parser import parse
+from itertools import izip
+from myria import MyriaConnection, MyriaError
+from schema import MyriaSchema
+
+class MyriaRelation(object):
+  DefaultConnection = MyriaConnection(hostname='localhost', port=8753)
+
+  # MyriaRelation({'userName': 'public', 'programName': 'adhoc', 'relationName': 'relation'})
+  # or
+  # MyriaRelation('public:adhoc:relation')
+  # or
+  # MyriaRelation('relation') # Defaults are 'public' and 'adhoc'
+  def __init__(self, relation, connection=DefaultConnection, *args, **kwargs):
```

Comment:
Why not a function that takes three arguments? If you want a shorthand, you could either have a builder or a named argument.